### PR TITLE
Parsing versions

### PR DESCRIPTION
**Pull request by mdugacek**
Closing #7

### CODE REVIEW

**Review by misoxxx — APPROVED** [review]
Ok, works good and code is clear.

File: src/parsers/parse_versions.cpp [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+#include <sparse/parsers/parsers.hpp>
+#include <unordered_set>
+
+namespace sparse::parsers
+{
+namespace detail
+{
+const static std::regex eal_regex("EAL[ ]?\\d[+]?");
+const static std::regex global_platform_regex("Global[ ]?Platform (\\d.\\d.\\d|\\d.\\d)");
+const static std::regex java_card_regex("Java Card (\\d.\\d.\\d|\\d)");
```
Comment (ondrys123) [inline src/parsers/parse_versions.cpp:12]: The dot character `.` matches "any character" instead of dot literal `\\.`  
or was this on purpose?

**Review by ondrys123 — COMMENTED** [review]
Implementation is fine, it would be nice to have some basic unit tests.

File: src/parsers/parse_versions.cpp (hunk 1 above)
Comment (mdugacek) [reply src/parsers/parse_versions.cpp]: No it was not. I will change that

**Review by ondrys123 — APPROVED** [review]
Looks good, in the future, if we want to test that vector contains something, we can use  
https://github.com/catchorg/Catch2/blob/devel/docs/matchers.md#vector-matchers

**Pull request merged**